arches_vue_utils/src/arches_vue_utils/utils.ts

```diff
@@ -54,11 +55,26 @@ export const getItemLabel = (
 ): Label => {
     const labels = (item as WithLabels).labels ?? (item as WithValues).values;
     if (!labels.length) {
-        throw new Error();
+        return {
+            value: "",
+            language_id: "",
+            valuetype_id: "",
+        };
     }
     return labels.sort(
         (a, b) =>
             rankLabel(b, preferredLanguageCode, systemLanguageCode) -
             rankLabel(a, preferredLanguageCode, systemLanguageCode),
     )[0];
 };
+
+export const getDescriptors = (
+        data: Descriptors,
+        systemLanguageCode: string,
+    ): [string, string] => {
+    if (!data) {
+        return ["None", "None"];
```

Comment:
Why not throw an error here? `["None", "None"]` seems like an odd thing to return

Reply:
I was assuming (perhaps wrongly) that my [suggestion](https://github.com/archesproject/arches-vue-utils/pull/5#discussion_r1852384080) in #5 would be implemented and was following that pattern. However, as it stands now, In getItemLabel above, we return empty string if the label isn't found. I would imagine that it's valid for resources to not always have descriptors (depending on the data model), so we should provide a fall-back value (whether "" or "None") rather than erroring?

Reply:
I guess a `"None"` string is a bit odd for me. I'd either expect this to be `null`, or an empty string. 

As far as if it's better to throw an error or not -- I'm not sure. I'm happy to move forward this fallback values and change later if it ends up biting us